lib/modules/manager/gradle/extract.ts

```diff
@@ -110,13 +110,13 @@ export async function extractAllPackageFiles(
       }
 
       dep.registryUrls = [
-        ...new Set([
-          ...defaultRegistryUrls,
-          ...(dep.registryUrls ?? []),
-          ...registryUrls,
-        ]),
+        ...new Set([...registryUrls, ...(dep.registryUrls ?? [])]),
       ];
 
+      if (!dep.registryUrls.length) {
+        dep.registryUrls = defaultRegistryUrls;
```

Comment:
Can this be skipped if maven datasource already contains a default?

Reply:
Yes, that should work. I guess you are referring to:
https://github.com/renovatebot/renovate/blob/793095c1a02c61bee36ddabeaa99586748c75b0e/lib/modules/datasource/maven/index.ts#L58

and 

https://github.com/renovatebot/renovate/blob/793095c1a02c61bee36ddabeaa99586748c75b0e/lib/modules/datasource/index.ts#L223-L225

Maven datasource is the only one supported by the Gradle manager, so these 3 lines should actually be redundant.

Reply:
Thanks for the hint!